src/test/java/org/folio/mr/api/MediatedRequestActionsApiTest.java

```diff
@@ -132,6 +135,7 @@ void mediatedRequestConfirmationForLocalInstanceAndItem() {
   @Test
   @SneakyThrows
   void mediatedRequestConfirmationForLocalInstanceAndRemoteItem() {
+    // given
```

Comment:
I think these messages do not make a lot of sense

Reply:
I found them useful in complex tests where I can't understand where preparation ends.
https://en.wikipedia.org/wiki/Given-When-Then